CRM/Contribute/BAO/Contribution.php

```diff
@@ -220,6 +220,9 @@ public static function add(&$params, $ids = array()) {
 
     //add Account details
     $params['contribution'] = $contribution;
+    if (empty($contributionID)) {
+      self::recordAlwaysAccountsReceivable($params);
```

Comment:
Let's try to be more consistent in names to help future developers. If the setting is always_post_to_accounts_receivable let's make this the same.